os/fs/smartfs/Kconfig

```diff
@@ -3,6 +3,14 @@
 # see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
 #
 
+config CONFIG_SMART_FS_SECURE
```

Comment:
NAME is wrong. It shoud be ```SMART_FS_SECURE```. But we already provide smartfs named ```FS_SMARTFS```. How about ```FS_SMARTFS_SECURE``` for same pattern?

Reply:
Shall I include it as an Smartfs option with flag CONFIG_SMART_SECURE?